api/src/main/java/org/pmiops/workbench/db/model/DbUser.java

```diff
@@ -449,4 +451,42 @@ public String generatePDName() {
             .substring(0, PD_UUID_SUFFIX_SIZE);
     return getUserPDNamePrefix() + "-" + randomString;
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
+
+    if (!(o instanceof DbUser)) return false;
+
+    DbUser dbUser = (DbUser) o;
+
+    return new EqualsBuilder()
+        .append(userId, dbUser.userId)
+        .append(disabled, dbUser.disabled)
+        .append(username, dbUser.username)
+        .append(contactEmail, dbUser.contactEmail)
+        .append(freeTierCreditsLimitDollarsOverride, dbUser.freeTierCreditsLimitDollarsOverride)
+        .append(firstSignInTime, dbUser.firstSignInTime)
+        .append(creationTime, dbUser.creationTime)
+        .append(givenName, dbUser.givenName)
+        .append(familyName, dbUser.familyName)
+        .append(address, dbUser.address)
+        .isEquals();
+  }
+
+  @Override
+  public int hashCode() {
```

Comment:
Here is how other equal and hascode looks like: https://github.com/all-of-us/workbench/blob/main/api/src/main/java/org/pmiops/workbench/db/model/DbInstitutionTierRequirement.java#L92L111
We can keep all in consistent. But I feel yours is better.. Up to you.

Reply:
thanks, yea I saw that, but I also felt the apache commons way is better.